modules/lazy-images/js/lazy-images.js

```diff
@@ -0,0 +1,134 @@
+/* globals IntersectionObserver, jQuery */
+
+/**
```

Comment:
Does the license for the polyfill need to go here? Also, I'd like to keep the polyfill in this same file to minimize network requests. It bails early if it detects `IntersectionObserver` is in `window`, so it shouldn't cause issues. If we leave it here though, should me unminify it?

Reply:
Just include the polyfill licence in Jetpack license, or somewhere like that. I don't think we need to serve it to the browser.